anet-dictionary.yml

```diff
@@ -1546,6 +1547,7 @@ fields:
     position:
       name: Afghan Tashkil
       type: Afghan Partner
+      positionRole: Position Role
```

Comment:
I would put this directly under `fields.position` since there's no need to have distinct labels for either advisor or principal.